src/util/enodetools.ts

```diff
@@ -35,72 +29,31 @@ export const enodeToParams = (enodeURL: string) => {
   } catch (err) {}
 
   return {
-    enodeHigh,
-    enodeLow,
+    enodeId,
     ip,
     port,
     extraParams
   };
 };
 
-export const paramsToIdentifier = ({
-  enodeHigh,
-  enodeLow,
-  ip,
-  port
-}: {
-  enodeHigh: string;
-  enodeLow: string;
-  ip: string;
-  port: string;
-}) => {
-  return `${enodeHigh}_${enodeLow}_${ip}_${port}`;
-};
-
-function parseHostname(stringHostname: string) {
-  if (stringHostname[0] === '[') {
-    const ipv6 = stringHostname.slice(1, -1);
-    return getHexIpv6(ipv6);
-  }
-  return getHexIpv4(stringHostname);
-}
-
-function getHexIpv4(stringIp: string) {
-  const splitIp = stringIp.split('.');
-  return `0x00000000000000000000ffff${toHex(splitIp[0])}${toHex(splitIp[1])}${toHex(splitIp[2])}${toHex(splitIp[3])}`;
-}
-
-function getHexIpv6(stringIpv6: string) {
-  const ipv6 = padIpv6(stringIpv6)
-    .split(':')
-    .join('');
-  return '0x' + ipv6;
-}
-
-function toHex(number: string) {
-  const num = Number(number).toString(16);
-  return leftPad(num, 2, '0');
-}
-
-export const buildEnode = (enodeHigh: string, enodeLow: string) => {
-  return `${enodeHigh.slice(2)}${enodeLow.slice(2)}`;
+export const paramsToIdentifier = ({ enodeId, ip, port }: { enodeId: string; ip: string; port: string }) => {
+  return `${enodeId}_${ip}_${port}`;
 };
 
 export const identifierToParams = (identifier: string) => {
-  const [enodeHigh, enodeLow, ip, port] = identifier.split('_');
+  const [enodeId, ip, port] = identifier.split('_');
   return {
-    enodeHigh,
-    enodeLow,
+    enodeId,
     ip,
     port,
     identifier
   };
 };
 
-export const identifierToEnodeHighAndLow = (identifier: string) => {
+export const identifierToEnodeId = (identifier: string) => {
```

Comment:
I believe we don't need this method anymore, do we? Wasn't it used to "glue" the two enodeId pieces into one?

Reply:
it still extracts just the Id part from Id_host_port